pylustrator.yaml

```diff
@@ -0,0 +1,5 @@
+repo: rgerum/pylustrator
+site: https://pylustrator.readthedocs.io/en/latest/
+keywords: interactive, gui, code generation
```

Comment:
We *need* the `section` field or it will just get put in misc...
```suggestion
keywords: interactive, gui, code generation
section: gui applications
```